2_map/src/data_map.R

```diff
@@ -41,20 +42,21 @@ boxplot_func <- function(regression_data, type, site_info)
       geom_violin() +
       stat_summary(fun = median, fun.min = median, fun.max = median,
                    geom = "crossbar",
-                   width = 0.25, color = 'red') +
-      #geom_jitter(shape=16, position=position_jitter(0.2), alpha = 0.4) +
-      geom_jitter(aes(shape=`Significant above a P of 0.01`), position=position_jitter(0.2), alpha = 0.4) +
-      scale_shape_manual(values = c(1,16)) +
-      # theme_bw(axis.text.x = element_text(angle = 90)) +
+                   width = 0.25, color = 'black') +
+      geom_jitter(fill = ifelse(regression_data$change_category == 'no change', "white", "grey"), aes(shape=`Significant above a P of 0.05`,
```

Comment:
You should be able to change this to `aes(fill = change_category)` and keep the same shape for all dots. Then you can do  `scale_fill_manual` to set your three different colors.